cmd/operator-sdk/bundle/cmd.go

```diff
@@ -33,15 +33,20 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "bundle",
 		Short: "Manage operator bundle metadata",
-		Long: `Manage bundle builds, bundle metadata generation, and bundle validation.
+		Long: `
```

Comment:
We should always follow this format in order to make the info be rendered cleared in the command line.